src/js/auth.test.js

```diff
@@ -112,7 +112,7 @@ describe('auth', () => {
       );
       expect(hasPermission([{ name: 'test' }], null, ['test'], 1)).toBe(true);
       expect(hasPermission([{ name: 'test' }], { id: 1 }, ['test'], null)).toBe(
-        false,
+        true,
```

Comment:
An org user should not have permission to query a non-org endpoint, which is what this function is checking.

Reply:
Makes sense.   I'd assumed only admin would have a null value for requestedOrgId.  Is there another way to id admin?

Reply:
Unfortunately, that part of the project is not very intuitive.
I wonder whether it would be cleaner to require every user to be assigned an org ID, with Greenstand as a top-level org and all others as child orgs.